apps/mark-scan/backend/src/app.test.ts

```diff
@@ -394,46 +398,47 @@ test('empty ballot box', async () => {
 });
 
 test('getPaperHandlerState returns state machine state', async () => {
-  await configureForTestElection();
+  await configureForTestElection(electionGeneralDefinition);
   await apiClient.setAcceptingPaperState();
   expect(await apiClient.getPaperHandlerState()).toEqual('accepting_paper');
 });
 
-test('setAcceptingPaperState is a no-op when USE_MOCK_PAPER_HANDLER flag is on', async () => {
```

Comment:
I removed this as a no-op but don't have a good understanding of why it was no-op, so may have to revert.